src/components/ButtonPanel.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropType from 'prop-types';
+import Button from './Button';
+
+const ButtonPanel = ({ id }) => (
```

Comment:
- ButtonPannel should not be taking props.

Reply:
Removed the id prop